dev-python/prov/prov-1.5.0.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
```

Comment:
How do I get python3.6 Also, I think eselect is confused, how do I make it understand that I actually don't have python3.5?

```
zenbookhost ~ # eix -e python
[U] dev-lang/python
     Available versions:  
     (2.7)  2.7.12 (~)2.7.13
     (3.4)  3.4.5(3.4/3.4m) (~)3.4.6(3.4/3.4m)
     (3.5)  (~)3.5.2(3.5/3.5m) (~)3.5.3(3.5/3.5m)
     (3.6)  (~)3.6.0(3.6/3.6m) (~)3.6.1(3.6/3.6m)
       {-berkdb build doc examples gdbm hardened ipv6 libressl +ncurses +readline sqlite +ssl +threads tk +wide-unicode wininst +xml ELIBC="uclibc"}
     Installed versions:  2.7.13(2.7)(11:22:28 21.02.2017)(gdbm ipv6 ncurses readline sqlite ssl threads wide-unicode xml -berkdb -build -doc -examples -hardened -libressl -tk -wininst ELIBC="-uclibc") 3.4.6(3.4)(11:25:20 21.02.2017)(gdbm ipv6 ncurses readline sqlite ssl threads xml -build -examples -hardened -libressl -tk -wininst ELIBC="-uclibc")
     Homepage:            https://www.python.org/
     Description:         An interpreted, interactive, object-oriented programming language

zenbookhost ~ # eselect python list
Available Python interpreters, in order of preference:
  [1]   python2.7
  [2]   python3.4
  [3]   python3.5
zenbookhost ~ # python3.5
-su: python3.5: command not found
```